src/webgpu/api/operation/rendering/depth_clip_clamp.spec.ts

```diff
@@ -468,9 +468,9 @@ to be empty.`
           depthClearValue: 1.0,
           depthLoadOp: 'clear',
           depthStoreOp: 'store',
-          stencilClearValue: 0,
-          stencilLoadOp: 'clear',
-          stencilStoreOp: 'discard',
+          stencilClearValue: info.stencil ? 0 : undefined,
```

Comment:
If you don't mind I'd rather keep the code a bit more verbose because it matches the other places we do this better and is more "regular". Feel free to ask for a follow-up though.